libraries/botbuilder/src/botFrameworkAdapter.ts

```diff
@@ -230,6 +230,11 @@ export class BotFrameworkAdapter extends BotAdapter {
         const client: ConnectorClient = this.createConnectorClient(reference.serviceUrl);
         const response = await client.conversations.createConversation(parameters);
 
+        // Mix in the tenant ID if specified. This is required for MS Teams.
+        if (reference.channelData && reference.channelData.tenant) {
```

Comment:
hey, @benbrown , it looks like this snipped of code is supposed to be above the `createConversation` call.
This is how I tweaked it to get it working as intended.
Cheers

Reply:
Thanks for catching this! Fixed!